elizaOS/eliza
# docs: fix Contributing Guide

# Relates to:


No specific issue. Minor documentation fix.

---

# Risks

Low.  
Fixes a broken link in the documentation. No functional changes.

---

# Background

## What does this PR do?

This PR fixes a broken link in the Contributing Guide section of `docs/docs/guides/local-development.md`.  
The link was pointing to `../community/contributing.md` and is now updated to `/contributing.md`.

## What kind of change is this?

Improvements:  
A minor change to existing documentation to fix a broken link.

---

# Documentation changes needed?

My changes do not require a change to the project documentation.

---

# Testing

## Where should a reviewer start?

Start by reviewing the modified file:  
`docs/docs/guides/local-development.md`.

## Detailed testing steps

1. Open the file `docs/docs/guides/local-development.md` in the rendered documentation.  
2. Click on the updated Contributing Guide link and verify it redirects correctly to `/contributing.md`.
